packages/modules/src/modules/banners/momentTemplate/MomentTemplateBanner.stories.tsx

```diff
@@ -125,3 +126,255 @@ WithReminder.args = {
+            'Fearless, investigative reporting shapes a fairer world. At the Guardian, our independence allows us to chase the truth wherever it takes us. We have no shareholders. No vested interests. Just the determination and passion to bring readers quality reporting, including groundbreaking investigations. We do not shy away. And we provide all this for free, for everyone.',
+        paragraphs: [
+            'Fearless, investigative reporting shapes a fairer world. At the Guardian, our independence allows us to chase the truth wherever it takes us. We have no shareholders. No vested interests. Just the determination and passion to bring readers quality reporting, including groundbreaking investigations.',
+            'We do not shy away. And we provide all this for free, for everyone.',
+        ],
+        highlightedText:
+            'Show your support today from just %%CURRENCY_SYMBOL%%1, or sustain us long term with a little more. Thank you.',
+        cta: {
+            text: 'Support us',
+            baseUrl: 'https://support.theguardian.com/contribute/one-off',
+        },
+        secondaryCta: {
+            type: SecondaryCtaType.Custom,
+            cta: {
+                text: 'Learn more',
+                baseUrl: 'https://support.theguardian.com/contribute/recurring',
+            },
+        },
+    },
+    numArticles: 50,
+    tickerSettings: undefined,
+};
+
+const BannerWithChoiceCards = bannerWrapper(
+    getMomentTemplateBanner({
+        containerSettings: {
+            backgroundColour: '#F1F8FC',
+        },
+        headerSettings: {
+            textColour: '#0077B6',
+        },
+        primaryCtaSettings: {
+            default: {
+                backgroundColour: '#0077B6',
+                textColour: 'white',
+            },
+            hover: {
+                backgroundColour: '#004E7C',
+                textColour: 'white',
+                border: '1px solid #004E7C',
+            },
+        },
+        secondaryCtaSettings: {
+            default: {
+                backgroundColour: '#F1F8FC',
+                textColour: '#004E7C',
+                border: '1px solid #004E7C',
+            },
+            hover: {
+                backgroundColour: '#E5E5E5',
+                textColour: '#004E7C',
+                border: '1px solid #004E7C',
+            },
+        },
+        closeButtonSettings: {
+            default: {
+                backgroundColour: '#F1F8FC',
+                textColour: brand[400],
+                border: `1px solid ${brand[400]}`,
+            },
+            hover: {
+                backgroundColour: '#E5E5E5',
+                textColour: brand[400],
+            },
+            theme: 'brand',
+        },
+        highlightedTextSettings: {
+            textColour: neutral[0],
+            highlightColour: brandAlt[400],
+        },
+        choiceCards: true,
+        bannerId: 'global-new-year-banner',
+    }),
+    'global-new-year-banner',
```

Comment:
Don't think it matters in storybook but should this banner name be different to that on line 193?

Reply:
No it doesn't matter in storybook and the real banner has not been added yet